packages/wallets/src/features/accounts/screens/ManualDocumentUpload/components/DocumentRuleHints/DocumentRuleHints.scss

```diff
@@ -0,0 +1,12 @@
+.wallets-document-rule-hints {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+
+    @include mobile {
+        display: grid;
```

Comment:
Is it ok to use flex for desktop and grid for mobile?

Reply:
there shouldn't be any problem right? 🤔